testsuite/integration-arquillian/servers/app-server/tomcat/tomcat7/pom.xml

```diff
@@ -26,7 +26,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>integration-arquillian-servers-app-server-tomcat7</artifactId>
-    <packaging>pom</packaging>
+    <packaging>jar</packaging>
     <name>App Server - Tomcat - Tomcat 7</name>
```

Comment:
I'd suggest `App Server - Tomcat X` as a name, it'd be consistent with other app-server modules (the same for both Tomcat 8 and Tomcat 9)